### PR TITLE
Fix Lint and Jest Failing

### PR DESCRIPTION
**Pull request by kelvinkipruto**
## Description

This PR fixes a bug that causes `lint` and `jest` to fail.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: .github/workflows/dependabot_lint_format.yml [hunk 1]
```diff
@@ -66,3 +66,9 @@ jobs:
         with:
           commit_message: Format & lint pnpm-lock.yaml
           file_pattern: "./pnpm-lock.yaml"
+
+      - name: "Run lint and jest if there are changes"
+        if: steps.auto-commit-action.outputs.changes_detected == 'true'
+        run: |
+          pnpm lint-check
+          pnpm test
```
Comment (kilemensi) [inline .github/workflows/dependabot_lint_format.yml:74]: 1. We're you going to add `pnpm build` as well?
2. What's the difference running them like this vs running them individually?
```yaml
      - name: "Lint if there are changes"
        if: steps.auto-commit-action.outputs.changes_detected == 'true'
        run: pnpm lint-check

      - name: "Test if there are changes"
        if: steps.auto-commit-action.outputs.changes_detected == 'true'
        run: pnpm test
```

File: packages/eslint-config-commons-ui/package.json [hunk 2]
```diff
@@ -27,7 +27,7 @@
     "eslint-plugin-jsx-a11y": "^6.7.1",
     "eslint-plugin-markdown": "^3.0.0",
     "eslint-plugin-playwright": "^0.12.0",
-    "eslint-plugin-prettier": "^5.0.0",
+    "eslint-plugin-prettier": "^4.2.1",
```
Comment (kilemensi) [inline packages/eslint-config-commons-ui/package.json:30]: Why?

**Review by kilemensi — COMMENTED** [review]
:+1:

File: packages/eslint-config-commons-ui/package.json (hunk 2 above)
Comment (kelvinkipruto) [reply packages/eslint-config-commons-ui/package.json:30]: This caused linting to fail.
We're using `prettier 2.8.8` and there seem to be some incompatibility issues based on https://github.com/prettier/eslint-plugin-prettier/issues/562.

Comment (kilemensi) [reply packages/eslint-config-commons-ui/package.json:30]: Why are we using prettier 2.x if 3.x is already out?

File: .github/workflows/dependabot_lint_format.yml [hunk 3]
```diff
@@ -66,3 +66,15 @@ jobs:
         with:
           commit_message: Format & lint pnpm-lock.yaml
           file_pattern: "./pnpm-lock.yaml"
+
+      - name: "Run lint there are changes"
```
Comment (kilemensi) [inline .github/workflows/dependabot_lint_format.yml:70]:
```suggestion
      - name: "Run lint (if there are changes)"
```
Emphasis on **if**

File: packages/eslint-config-commons-ui/package.json (hunk 2 above)
Comment (kilemensi) [reply packages/eslint-config-commons-ui/package.json:30]: How is this resolved?

**Comment by kelvinkipruto** [comment]
@kilemensi Upgrading to `prettier 3` introduces many changes when linting is done. Lets do that on a separate PR

**Comment by kilemensi** [comment]
> @kilemensi Upgrading to `prettier 3` introduces many changes when linting is done. Lets do that on a separate PR

Exactly, you're suppose to _resolve_ the issue before marking it as **resolved** @kelvinkipruto :
1. Create an issue to track upgrade of `prettier` to v3,
2. Attach it to that convo, and then
3. Mark the convo as resolved.

File: packages/eslint-config-commons-ui/package.json (hunk 2 above)
Comment (kelvinkipruto) [reply packages/eslint-config-commons-ui/package.json:30]: This will be resolved in #527

**Review by kilemensi — APPROVED** [review]
🚀

**Pull request merged**